app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -109,6 +109,19 @@ def show_delete
 
     private
 
+    def get_browser_agent
```

Comment:
as a style thing, we typically don't name methods in ruby with `get_`, I'd just call it `browser_agent` or `formatted_browser_agent` if we want to be clear

Reply:
👍 thank you!